dags/bqetl_artifact_deployment.py

```diff
@@ -135,7 +136,18 @@
         image=docker_image,
     )
 
+    publish_static_tables = GKEPodOperator(
+        task_id="publish_static_tables",
+        cmds=["bash", "-x", "-c"],
+        arguments=[
+            "script/bqetl static publish --project_id=moz-fx-data-shared-prod && "
+            "script/bqetl static publish --project_id=mozdata"
+        ],
+        image=docker_image,
+    )
+
     publish_views.set_upstream(publish_public_udfs)
     publish_views.set_upstream(publish_persistent_udfs)
     publish_views.set_upstream(publish_new_tables)
     publish_metadata.set_upstream(publish_views)
+    publish_metadata.set_upstream(publish_static_tables)
```

Comment:
IMO it'd be best for the `publish_metadata` task to directly depend on all the relevant tasks (including `publish_new_tables`), rather than relying on `publish_views`' dependencies second-hand.
```suggestion
    publish_metadata.set_upstream(publish_new_tables)
    publish_metadata.set_upstream(publish_static_tables)
```